mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -56,7 +56,10 @@ private[ml] trait CrossValidatorParams extends ValidatorParams {
 
 /**
  * :: Experimental ::
- * K-fold cross validation.
+ * CrossValidator begins by splitting the dataset into a set of non-overlapping randomly
```

Comment:
sorry about this, but looking at it again I think we can clean up the description a bit more:

`CrossValidator performs model selection by splitting the dataset into a set of non-overlapping randomly partitioned folds, which are used as separate training and test datasets. For example, with k=3 folds, ...`